src/components/CouponDetails/index.jsx

```diff
@@ -403,6 +407,18 @@ class CouponDetails extends React.Component {
       tableColumns.splice(getColumnIndexForKey('assigned_to'), 1);
     }
 
+    // `assignments_remaining` column
+    if (value === 'unassigned' && getColumnIndexForKey('assignments_remaining') === -1) {
+      // Add `assignments_remaining` column if it doesn't already exist.
+      tableColumns.splice(3, 0, {
+        label: 'Assignments Remaining',
+        key: 'assignments_remaining',
+      });
+    } else if (value !== 'unassigned' && getColumnIndexForKey('assigned_to') > -1) {
```

Comment:
Are you meaning to use `assigned_to` here versus `assignments_remaining`?

Reply:
Oh, yeah. Nice catch ;)

Reply:
Once that's fixed up, I'll give a 👍!